### PR TITLE
PLG-681: remove CVM feature flag

### PR DESCRIPTION
**Pull request by antoinetrouve**
Cleanup the code to use only DSM Catalog Volume Monitoring view.

**Description (for Contributor and Core Developer)**

**Definition Of Done (for Core Developer only)**

- [ ] Tests
- [ ] Migration & Installer
- [ ] PM Validation (Story)
- [ ] Changelog (maintenance bug fixes)
- [ ] Tech Doc

### CODE REVIEW

**Comment by LaurentPetard** [comment]
There also the old dead code to remove. Such as https://github.com/akeneo/pim-community-dev/tree/master/src/Akeneo/Platform/Bundle/UIBundle/Resources/public/js/catalog-volume
But it can be done in another PR

**Comment by antoinetrouve** [comment]
> There's also the old dead code to remove. Such as https://github.com/akeneo/pim-community-dev/tree/master/src/Akeneo/Platform/Bundle/UIBundle/Resources/public/js/catalog-volume But it can be done in another PR

I did it on the last commit :  https://github.com/akeneo/pim-community-dev/pull/15871/commits/a1408c11715d337496929d0df4e5cbfa664b7be7

File: src/Akeneo/Platform/Bundle/UIBundle/Resources/config/form_extensions/catalog_volume/index.yml [hunk 1]
```diff
@@ -57,14 +47,6 @@ extensions:
...
 
-    pim-catalog-volume-product-values-section-content:
-        module: pim/catalog-volume/section
-        parent: pim-catalog-volume-product-values-section
-        targetZone: content
-        config:
-            align: right
-            axes: [count_products, count_channels, count_scopable_attributes, average_max_attributes_per_family, count_locales, count_localizable_and_scopable_attributes, count_localizable_attributes]
-
     pim-catalog-volume-catalog-structure-section:
```
Comment (olivierpontier) [inline src/Akeneo/Platform/Bundle/UIBundle/Resources/config/form_extensions/catalog_volume/index.yml:50]: This section should not used anymore. We can remove it

File: src/Akeneo/Platform/Bundle/UIBundle/Resources/config/form_extensions/catalog_volume/index.yml [hunk 2]
```diff
@@ -76,13 +58,6 @@ extensions:
...
                 title: pim_catalog_volume.section.catalog_structure.hint
 
-    pim-catalog-volume-catalog-structure-section-content:
-        module: pim/catalog-volume/section
-        parent: pim-catalog-volume-catalog-structure-section
-        targetZone: content
-        config:
-            axes: [count_families, count_attributes, average_max_options_per_attribute, count_categories, count_category_trees]
-
     pim-catalog-volume-variant-modelling-section:
```
Comment (olivierpontier) [inline src/Akeneo/Platform/Bundle/UIBundle/Resources/config/form_extensions/catalog_volume/index.yml:61]: This section should not used anymore. We can remove it

File: src/Akeneo/Platform/Bundle/UIBundle/Resources/config/form_extensions/catalog_volume/index.yml [hunk 3]
```diff
@@ -35,16 +35,6 @@ extensions:
...
-        parent: pim-catalog-volume-index
-        module: pim/catalog-volume/header
-        targetZone: content
-        position: 10
-        config:
-            title: pim_catalog_volume.product_values
-            description: pim_catalog_volume.product_values_description
-            link: https://medium.com/akeneo-labs/because-your-product-catalog-typology-matters-e1a9af4c33e0
-
     pim-catalog-volume-product-values-section:
```
Comment (olivierpontier) [inline src/Akeneo/Platform/Bundle/UIBundle/Resources/config/form_extensions/catalog_volume/index.yml:38]: This section should not used anymore. We can remove it

**Comment by olivierpontier** [comment]
The whole form extension configuration for the Catalog Volume Monitoring page seems to not be useful anymore: https://github.com/akeneo/pim-community-dev/blob/master/src/Akeneo/Platform/Bundle/UIBundle/Resources/config/form_extensions/catalog_volume/index.yml
Coud you confirm that we do not need it anymore and remove it, please?

**Review by olivierpontier — APPROVED** [review]
Good to merge after ensuring the form extension is not used anymore.
It is very pleasant to seea PR for removing "legacy code" :)

**Review by BryanS29-akeneo — APPROVED** [review]

**Comment by antoinetrouve** [comment]
> Good to merge after ensuring the form extension is not used anymore. It is very pleasant to see a PR for removing "legacy code" :)

Effectively, the whole catalog volume form extension is not used anymore. There was still a reference to this one :  _pim-catalog-volume-index_  in _public/js/controller/catalog-volume.tsx::renderForm()_. But this function was never called. So i remove it too.

File: src/Akeneo/Platform/Bundle/UIBundle/Resources/config/form_extensions/catalog_volume/index.yml (hunk 2 above)
Comment (antoinetrouve) [reply src/Akeneo/Platform/Bundle/UIBundle/Resources/config/form_extensions/catalog_volume/index.yml]: Done in my last commit

File: src/Akeneo/Platform/Bundle/UIBundle/Resources/config/form_extensions/catalog_volume/index.yml (hunk 3 above)
Comment (antoinetrouve) [reply src/Akeneo/Platform/Bundle/UIBundle/Resources/config/form_extensions/catalog_volume/index.yml]: Done in my last commit

File: src/Akeneo/Platform/Bundle/UIBundle/Resources/config/form_extensions/catalog_volume/index.yml (hunk 1 above)
Comment (antoinetrouve) [reply src/Akeneo/Platform/Bundle/UIBundle/Resources/config/form_extensions/catalog_volume/index.yml]: Done too

**Review by LaurentPetard — APPROVED** [review]

**Pull request merged**